hedera-node/src/main/java/com/hedera/services/fees/calculation/crypto/queries/GetAccountDetailsResourceUsage.java

```diff
@@ -0,0 +1,85 @@
+package com.hedera.services.fees.calculation.crypto.queries;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.context.primitives.StateView;
+import com.hedera.services.context.properties.GlobalDynamicProperties;
+import com.hedera.services.fees.calculation.QueryResourceUsageEstimator;
+import com.hedera.services.ledger.accounts.AliasManager;
+import com.hedera.services.usage.crypto.CryptoOpsUsage;
+import com.hedera.services.usage.crypto.ExtantCryptoContext;
+import com.hederahashgraph.api.proto.java.FeeData;
+import com.hederahashgraph.api.proto.java.Query;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Map;
+
+@Singleton
+public final class GetAccountDetailsResourceUsage implements QueryResourceUsageEstimator {
```

Comment:
Since you've already done the work, can leave it---but `0.0.2` and `0.0.50` aren't actually charged for queries! 😜

Reply:
Ohhh 👀